Tests/ExportDumpTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Cybex\Protector\Exceptions\FailedCreatingDestinationPathException;
+use Cybex\Protector\Exceptions\FailedMysqlCommandException;
+use Cybex\Protector\Exceptions\InvalidConnectionException;
+use Cybex\Protector\Protector;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Storage;
+
+class ExportDumpTest extends BaseTest
+{
+    /**
+     *  Protector instance.
+     */
+    protected Protector $protector;
+
+    protected string $baseDirectory;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Config::set('protector.baseDirectory', 'protector');
+
+        $this->protector     = app('protector');
+        $this->disk          = Storage::disk('local');
```

Comment:
Dynamic properties will be deprecated in PHP 8.2 and will raise a warning. They will will throw an exception in PHP 9. Might be the best to declare the properties in the class across all tests.

Reply:
Done.